python/paddle/fluid/tests/unittests/test_directory_migration.py

```diff
@@ -0,0 +1,170 @@
+            'paddle.imperative.BackwardStrategy',
+            'paddle.imperative.ParallelEnv',
+            'paddle.imperative.prepare_context',
+            'paddle.imperative.DataParalell', 'paddle.imperative.jit',
+            'paddle.imperative.TracedLayer', 'paddle.imperative.declarative',
+            'paddle.imperative.ProgramTranslator',
+            'paddle.imperative.TranslatedLayer', 'paddle.imperative.jit.save',
+            'paddle.imperative.jit.load',
+            'paddle.imperative.jit.SaveLoadConfig',
+            'paddle.imperative.NoamDecay'
+            'paddle.imperative.PiecewiseDecay',
+            'paddle.imperative.NaturalExpDecay',
+            'paddle.imperative.ExponentialDecay',
+            'paddle.imperative.InverseTimeDecay',
+            'paddle.imperative.PolynomialDecay',
+            'paddle.imperative.CosineDecay', 'paddle.Executor',
+            'paddle.global_scope', 'paddle.scope_guard',
+            'paddle.append_backward', 'paddle.gradients',
+            'paddle.BuildStrategy', 'paddle.CompiledProgram',
+            'paddle.ExecutionStrategy', 'paddle.name_scope',
+            'paddle.program_guard', 'paddle.Print', 'paddle.py_func',
+            'paddle.ParallelExecutor', 'paddle.default_main_program',
+            'paddle.default_startup_program', 'paddle.Program',
+            'paddle.WeightNormParamAttr', 'paddle.declarative.fc',
+            'paddle.declarative.batch_norm',
+            'paddle.declarative.bilinear_tensor_product',
+            'paddle.declarative.conv2d', 'paddle.declarative.conv2d_transpose',
+            'paddle.declarative.conv3d', 'paddle.declarative.conv3d_transpose',
+            'paddle.declarative.create_parameter',
+            'paddle.declarative.crf_decoding', 'paddle.declarative.data_norm',
+            'paddle.declarative.deformable_conv',
+            'paddle.declarative.group_norm', 'paddle.declarative.hsigmoid',
+            'paddle.declarative.instance_norm', 'paddle.declarative.layer_norm',
+            'paddle.declarative.multi_box_head', 'paddle.declarative.nce',
+            'paddle.declarative.prelu', 'paddle.declarative.row_conv',
+            'paddle.declarative.spectral_norm', 'paddle.declarative.embedding'
+        ]
+
+        import_file = 'run_old_import_modules.py'
+
+        with open(import_file, "w") as wb:
+            wb.write('count = 0\n')
+            wb.write('err_module = ""\n')
+            for module in old_directory:
+                run_cmd = self.get_import_command(module)
+                wb.write('try:\n')
+                wb.write('    ' + run_cmd + '\n')
+                wb.write('except:\n')
+                wb.write('    count += 1\n')
+                wb.write('else:\n')
+                wb.write('    err_module = "' + module + '"\n')
+            wb.write('if count != ' + str(len(old_directory)) + ':\n')
+            wb.write('    print("Error: Module " + err_module +' \
```

Comment:
It's ok to write these lines, while I suggest you refine these lines by `string formating` for better readability in next PR. (maybe you already know that)
For example,

```
try_cmd_template = '''
try:
    {cmd}
except:
    count += 1
else:
    err_module={module}
'''

try_cmd = try_cmd_template.format(cmd="import paddle.xx", module="paddle.xx")
```

Reply:
This format has been implemented in this PR.